src/main/java/com/pnudev/librarysystem/security/UserDetailsImpl.java

```diff
@@ -12,7 +12,7 @@
 
 @AllArgsConstructor
 public class UserDetailsImpl implements UserDetails {
-
+    private Long id;
```

Comment:
Consider adding a `@Setter` above this field

Reply:
Done